packages/server/assets/fileuploads/typedefs/fileuploads.graphql

```diff
@@ -95,6 +95,11 @@ type FileUpload {
   Model associated with the file upload, if it exists already
   """
   model: Model
+  modelId: String
+  """
+  Date when upload was last updated
+  """
+  updatedAt: DateTime!
```

Comment:
confusing that there's both lastConvertedAt and lastUploadedAt - as a consumer I virtually always just want the latest updatedAt